market/genderize/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+	"__comment": "prototype for first connector",
+
+	"name": "Genderize.io",
+	"listed": false,
+
+	"installer": {
+		"id": "open-api-rest-client",
+		"properties": {
+			"openApiUrl": "https://raw.githubusercontent.com/axonivy/market/master/market/genderize/openapi.json",
```

Comment:
I think, if we commit the open api spec to this repository, we may should point it then relative.
(+) makes life easier for the maintainer of this repository. you can change the folder structure.
(+) makes life easier for connector provider, you don't need to know the "end" url.
(-) need to implement a relative path resolver in the open-api-rest-client installer.

good for now. in future I will implement a relative path resolver. anyway done for loading the logo.png.

Reply:
Yes, we definitively will need a relative path resolver infrastructure.
For the future I could also imagine to create new repositories for each and every connector where we provide openapi.json ... with the idea to drive the openapi initiative in a way where others could contribute and enrich the specs. 
Like in this scenario here: I think for the operators of genderize.io it would be an easy fish to maintain this spec. And I think they would do that, if this repo does not live in our 'market' environment.